cosmo_tester/resources/attributes.yaml

```diff
@@ -4,6 +4,10 @@ cloudify_username: admin
 cloudify_password: admin
 cloudify_tenant: default_tenant
 
+# If the latest image name is not set, the latest image name will be generated
+# based on the image name prefix and the currently installed CLI version
+# e.g. cloudify-manager-premium-4.2.dev1
+cloudify_manager_latest_image: ''
```

Comment:
It... kinda isn't (it's the one for the currently installed CLI), but the name of the function was _get_latest(etc) so I was maintaining that.

Do you think it's worth doing something about the naming here or is it worth adding comments to https://cloudifysource.atlassian.net/browse/CFY-7159 ?